spec/features/two_factor_authentication/change_factor_spec.rb

```diff
@@ -15,10 +15,39 @@
     end
 
     scenario 'editing phone number' do
+      allow(SmsSenderNumberChangeJob).to receive(:perform_later)
+
+      @previous_phone_confirmed_at = @user.phone_confirmed_at
+
       visit edit_phone_path
       complete_2fa_confirmation
 
-      expect(current_path).to eq edit_phone_path
+      # update the phone number
```

Comment:
is this commenting a style we are using throughout this test suite? I haven't noticed it. A style I prefer is to put stuff like this into named methods, eg `update_phone_number`